src/ai/backend/manager/models/gql_models/image.py

```diff
@@ -314,10 +324,14 @@ class Meta:
         interfaces = (AsyncNode,)
 
     row_id = graphene.UUID(description="Added in 24.03.4. The undecoded id value stored in DB.")
-    name = graphene.String()
+    name = graphene.String(deprecation_reason="Deprecated since 24.12.0. use `namespace` instead")
+    namespace = graphene.String(description="Added in 24.12.0..")
+    base_image_name = graphene.String(description="Added in 24.12.0..")
```

Comment:
```suggestion
    base_image_name = graphene.String(description="Added in 24.12.0.")
```
And it's the same below.